packages/atomic/package.json

```diff
@@ -36,8 +36,11 @@
     "test:watch": "stencil test --spec --watchAll",
     "generate": "stencil generate",
     "cypress:test": "cypress run --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser chrome",
-    "cypress:test:firefox": "cypress --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser firefox",
+    "cypress:test:firefox": "cypress run --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser firefox",
     "cypress:open": "cypress open",
+    "cypress-hsp:test": "cypress run --config-file cypress-hsp.json --browser chrome",
```

Comment:
Should we add a line in github actions to call this script? Or will that be done in a separate PR/different way?

Reply:
The script is triggered at line 9 of [hosted-search-page.cypress.ts](https://github.com/coveo/ui-kit/pull/1702/files#diff-a39dff766e5636dbd454266545e1cb6ed65cc003bd68a950e68d3f5d8edf6238)
using cy.exec()

Reply:
Oh, I thought we were going to run them as a separate step. Including it as part of the atomic test suite work stoo ✔️

Reply:
To be clear, the flow now is like this:
npm run cypress:test --> run the usual tests, not the hosted search page test
npm run cypress-hsp:test --> run only the hosted search page test, with the download script being launched at the beginning of the test

Reply:
Just updated the README

Reply:
Ok, so we're back to the start of the thread :) Where does `npm run cypress-hsp:test` get called? Should it be run as part of ui-kit's continuous integration?